projects/storefrontlib/shared/components/hierarchy/hierarchy-node/hierarchy-node.component.html

```diff
@@ -0,0 +1,23 @@
+<div>
```

Comment:
Lets use css to apply the `display: block` style. No need to add further nesting to the DOM.